integration/integration.go

```diff
@@ -176,6 +176,20 @@ func (i *Integration) Logger() log.Logger {
 	return i.logger
 }
 
+// FindEntity finds ad return an entity by name. returns false if entity does not exist in the integration
+func (i *Integration) FindEntity(name string) (*Entity, bool) {
+	for _, e := range i.Entities {
+		if e.Metadata == nil {
+			return &Entity{}, false
```

Comment:
shouldn't be a continue here ? just to avoid checking a nil pointer in the following if

Reply:
yes you're right we need a continue to keep searching..late friday code :)